nominations/templates/email/application_submit_email.txt

```diff
@@ -1,19 +1,24 @@
-Hey Our Revolution Friend!
+** [OUR REVOLUTION] 
+------------------------------------------------------------
 
-We're excited to review your candidate nomination. While we cannot endorse everyone, we deeply appreciate all the care our groups take in sending us endorsed candidates.
+DEAR OUR REVOLUTION MEMBER,
```

Comment:
Why are we using all caps here? This seems odd.

Reply:
Automatically generated from Mailchimp's html to txt convertor, because this is a <strong> tag.

Reply:
\<strong> tag